plugin/trino-hive/src/test/java/io/trino/plugin/hive/s3select/TestIonSqlQueryBuilder.java

```diff
@@ -54,32 +56,46 @@
     @Test
     public void testBuildSQL()
     {
-        IonSqlQueryBuilder queryBuilder = new IonSqlQueryBuilder(TESTING_TYPE_MANAGER);
         List<HiveColumnHandle> columns = ImmutableList.of(
                 createBaseColumn("n_nationkey", 0, HIVE_INT, INTEGER, REGULAR, Optional.empty()),
                 createBaseColumn("n_name", 1, HIVE_STRING, VARCHAR, REGULAR, Optional.empty()),
                 createBaseColumn("n_regionkey", 2, HIVE_INT, INTEGER, REGULAR, Optional.empty()));
 
+        // CSV
+        IonSqlQueryBuilder queryBuilder = new CsvIonSqlQueryBuilder(TESTING_TYPE_MANAGER);
         assertEquals("SELECT s._1, s._2, s._3 FROM S3Object s",
                 queryBuilder.buildSql(columns, TupleDomain.all()));
+
         TupleDomain<HiveColumnHandle> tupleDomain = withColumnDomains(ImmutableMap.of(
                 columns.get(2), Domain.create(SortedRangeSet.copyOf(BIGINT, ImmutableList.of(Range.equal(BIGINT, 3L))), false)));
         assertEquals("SELECT s._1, s._2, s._3 FROM S3Object s WHERE (case s._3 when '' then null else CAST(s._3 AS INT) end = 3)",
                 queryBuilder.buildSql(columns, tupleDomain));
+
+        // JSON
+        queryBuilder = new JsonIonSqlQueryBuilder(TESTING_TYPE_MANAGER);
+        assertEquals("SELECT s.n_nationkey, s.n_name, s.n_regionkey FROM S3Object s",
```

Comment:
Please change the order. `assertEquals(actual, expected)` (here and in other places)

Reply:
Done, thanks for spotting.